app/models/post.rb

```diff
@@ -1,4 +1,8 @@
 class Post < ApplicationRecord
   has_many :likes
   has_many :comments, dependent: :destroy
+
+  scope :most_likes, ->{where(like_quantity: self.maximum(:like_quantity))}
```

Comment:
```suggestion
  scope :most_likes, ->{where like_quantity: self.maximum(:like_quantity)}
```
tương tự cho chỗ khác nhé